srcpkgs/ruby-backports/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'ruby-backports'
+pkgname=ruby-backports
+version=3.11.4
+revision=1
+noarch=yes
+build_style=gem
+short_desc="Essential backports that enable many of the nice features of Ruby"
+maintainer="maxice8 <thinkabit.ukim@gmail.com>"
+license="MIT"
+homepage="http://github.com/marcandre/backports"
```

Comment:
Available in https

Reply:
I'm using the homepage from their rubygems API